src/test/java/com/google/jenkins/plugins/k8sengine/KuberentesVerifiersTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.jenkins.plugins.k8sengine;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.io.Resources;
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.JsonPath;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class KuberentesVerifiersTest {
+  private static final Logger LOGGER = Logger.getLogger(KubernetesVerifiers.class.getName());
+
+  private static final String VERIFIABLE_DEPLOYMENT_OUTPUT = "verifiableDeploymentOutput.json";
+  private static final String UNVERIFIABLE_DEPLOYMENT_OUTPUT = "unverifiableDeploymentOutput.json";
+
+  private static KubectlWrapper kubectl;
+  private static Object goodDeploymentOutput;
+
+  private static Object badDeploymentOutput;
+
+  @BeforeClass
+  public static void init() throws IOException, InterruptedException {
+
+    LOGGER.info("Initializing KubernetesVerifiersTest");
+
+    // Configure mocked deployments here
+    kubectl = Mockito.mock(KubectlWrapper.class);
```

Comment:
Suggest avoiding shared setup code. It detracts from test readability and opens the door for parallelization issues.